pulp_rpm/app/migrations/0026_checksum_null.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.16 on 2020-10-09 14:58
+
+from django.db import migrations, models
+
+
+def check_checksum(apps, schema_editor):
+    checksum_model = apps.get_model("rpm", "Checksum")
+    q_bad_dist_tree = checksum_model.objects.filter(checksum=None)
+    if q_bad_dist_tree:
+        dist_tree_href = {
+            f"/pulp/api/v3/content/rpm/distribution_trees/{c.distribution_tree.pk}/"
+            for c in q_bad_dist_tree
+        }
+        raise ValueError(
+            "Your Pulp instance contains DistributionTree content units with null checksums."
```

Comment:
This message confuses me - if the distribution tree is used in at least one repo, the user won't be able to remove it.
If it is not present in any of the repos means it's an orphan then it should be removed by orphan clean up.
This change explicitly disables a user to remove dist_tree https://github.com/pulp/pulp_rpm/pull/1862/files

Reply:
You are right, to be user friendly we should point user to remove repository and run orphan cleanup not to remove dist tree.

Reply:
`Please remove the affected DistributionTree content from your repositories, run orphan cleanup and continue the upgrade.`
Is that better?

Reply:
it is, however i think we should also provide list of repositories the distribution trees are present in, thoughts?

Reply:
I agree with a repository list.